source/app/views/student_dashboard/_navbar.html.erb

```diff
@@ -2,6 +2,15 @@
   <div class="nav-signedin-logo">
     <%= image_tag 'real ratings.svg', alt: 'Real Rating Logo', class: 'd-inline-block align-text-top', size: '200x40' %>
   </div>
+          
+  <div class="profile-picture">
+    <% if current_user.profile_picture.attached? %>
+      <%= image_tag current_user.profile_picture, alt: "#{current_user.first_name} #{current_user.last_name}", class: 'profile-picture-img', size: '80x80' %>
+    <% else %>
+      <%= image_tag 'default_pfp.png', alt: 'Default Profile Picture', class: 'profile-picture-img', size: '80x80' %>
+    <% end %>
+  </div>
+
```

Comment:
This is just a suggestion but if you have time, could you make this pfp icon in the navbar link directly to the settings page when clicked? You could do something like:
``` ruby
<%= link_to settings_instructor_dashboard_index_path(course_id: @selected_course.id), method: :patch do %>
    <% if current_user.profile_picture.attached? %>
      <%= image_tag current_user.profile_picture, alt: "#{current_user.first_name} #{current_user.last_name}", class: 'profile-picture-img', size: '80x80' %>
    <% else %>
      <%= image_tag 'default_pfp.png', alt: 'Default Profile Picture', class: 'profile-picture-img', size: '80x80' %>
    <% end %>
<% end %>
```